File: sources/Silent 2.2.1/net/silentclient/client/utils/RandomUtils.java
package net.silentclient.client.utils;

import java.security.SecureRandom;

import io.netty.util.internal.ThreadLocalRandom;

public class RandomUtils {
	public static int randomInt(int min, int max) {
		return ThreadLocalRandom.current().nextInt(min, max + 1);
	}
	public static String randomString(int len)
    {
        final String chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
 
        SecureRandom random = new SecureRandom();
        StringBuilder sb = new StringBuilder();
 
        for (int i = 0; i < len; i++)
        {
            int randomIndex = random.nextInt(chars.length());
            sb.append(chars.charAt(randomIndex));
        }
 
        return sb.toString();
    }
}

File: sources/Raven B4 Remake/net/minusmc/ravenb4/utils/RandomUtils.kt
package net.minusmc.ravenb4.utils

import kotlin.random.Random

object RandomUtils {
	fun nextString(length: Int) {
		val chars = ('a'..'z') + ('0'..'9')
		return (1..length).map{chars.random()}.joinToString("")
	}

	fun nextInt(minimum: Int, maximum: Int) = Random.nextInt(maximum - minimum + 1) + minimum
}